Tasks/AzureAppServiceManageV0/package.json

```diff
@@ -22,7 +22,7 @@
     "@types/q": "1.0.7",
     "agent-base": "6.0.2",
     "azure-pipelines-task-lib": "^4.11.0",
-    "azure-pipelines-tasks-azure-arm-rest": "3.259.1",
+    "azure-pipelines-tasks-azure-arm-rest": "^3.259.2",
```

Comment:
Why here it's ^x.y.z, and in other place it's just x.y.z. Let's use just x.y..z - without '^'. So no regressions will come with latest versions and only hardcoded version will be used